pkg/adafruitio/adafruitio.go

```diff
@@ -78,6 +81,74 @@ func (c *Client) Record(feed, value string, timestamp time.Time) error {
 	return nil
 }
 
+// Feeds returns all feeds of an Adafruit user.
```

Comment:
[nit] Feeds _requests_ metadata on all feeds of an Adafruit user.

All functions return things. Comments should explain _how_ values are returned. For example, is there caching involved here?